schemas/string_interpolation/definitions.json

```diff
@@ -152,5 +166,42 @@
     },
     "additionalProperties": false,
     "required": ["source", "identifier"]
+  },
+  "text_plural": {
+    "description": "A mapping of a count to plural strings",
+    "type": "object",
+    "properties": {
+      "forms": {
+        "description": "Plural forms of the text. Uses the CLDR plural categories. One or more of these properties will be present dependent on the language.",
+        "type": "object",
+        "properties": {
```

Comment:
These should be under `"items"` with a ` "minItems": 2`.

Reply:
It's not an array so no need for items, but the `additionalProperties` and `required` properties are at the wrong level so I'll move them.